aggregator/src/test/java/com/redhat/cloud/notifications/db/EmailAggregationRepositoryTest.java

```diff
@@ -59,7 +65,7 @@ void afterEach() {
 
     @ParameterizedTest
     @ValueSource(booleans = {true, false})
-    void testApplicationsWithPendingAggregationAccordingOrgPref(boolean useSystemEndpoint) {
+    void testApplicationsWithPendingAggregationUsingDefaultSystemIntegration(boolean useSystemEndpoint) {
```

Comment:
Should this just be changed to an internal test method? Since it already gets called by the new `testAggregationWithSubscribersOnly` with both booleans